contracts/VotingMachines/AbsoluteVote.sol

```diff
@@ -31,12 +31,8 @@ contract AbsoluteVote is IntVoteInterface {
         bool open; // voting open flag
     }
 
-    event NewProposal(bytes32 indexed _proposalId, uint _numOfChoices, address _proposer, bytes32 _paramsHash);
-    event CancelProposal(bytes32 indexed _proposalId);
-    event ExecuteProposal(bytes32 indexed _proposalId, uint _decision, uint _totalReputation);
-    event VoteProposal(bytes32 indexed _proposalId, address indexed _voter, uint _vote, uint _reputation, bool _isOwnerVote);
-    event CancelVoting(bytes32 indexed _proposalId, address indexed _voter);
-    event RefreshReputation(bytes32 indexed _proposalId, address indexed _voter,uint _reputation);
+    event AVVoteProposal(bytes32 indexed _proposalId, address indexed _avatar, address indexed _voter, uint _vote, uint _reputation, bool _isOwnerVote);
```

Comment:
AVVoteProposal should include only the extended param which is not included at the interface event (plus the proposalId as Index)..
There is no need to duplicate .

Reply:
Done

Reply:
Why the _avatar  and _voter are needed indexes at the AVVoteProposal ?
One can fetch the proposalId for VoteProposal (by _avatar or _voter) and then look for AVVoteProposal according to _proposalId .
These two events are emitted one after the other.

Reply:
@orenyodfat  This is for API consistency.  Is there a cost to having this indexed _avatar here?